### ISSUE TITLE
MeshCentral restarts every time a device is renamed

### ISSUE THREAD

**Issue by whalehub** [Ylianst/MeshCentral#567]
This problem started to appear after I upgraded to the latest version (v0.4.2-h). Here's the output from `mesherrors.txt`:

````
-------- 10/13/2019, 12:44:27 PM ---- 0.4.2-h --------

/opt/meshcentral/node_modules/mongodb/lib/utils.js:133
      throw err;
      ^

ReferenceError: device is not defined
    at /opt/meshcentral/node_modules/meshcentral/meshuser.js:2309:67
    at /opt/meshcentral/node_modules/meshcentral/db.js:639:79
    at result (/opt/meshcentral/node_modules/mongodb/lib/operations/execute_operation.js:75:17)
    at executeCallback (/opt/meshcentral/node_modules/mongodb/lib/operations/execute_operation.js:68:9)
    at handleCallback (/opt/meshcentral/node_modules/mongodb/lib/utils.js:129:55)
    at cursor.close (/opt/meshcentral/node_modules/mongodb/lib/operations/to_array.js:36:13)
    at handleCallback (/opt/meshcentral/node_modules/mongodb/lib/utils.js:129:55)
    at completeClose (/opt/meshcentral/node_modules/mongodb/lib/cursor.js:859:16)
    at Cursor.close (/opt/meshcentral/node_modules/mongodb/lib/cursor.js:878:12)
    at cursor._next (/opt/meshcentral/node_modules/mongodb/lib/operations/to_array.js:35:25)
```

**Comment by yakatz** [comment]
Same error for me.

**Comment by yakatz** [comment]
I have to go back to v0.4.2-e before this works again.

**Comment by Ylianst** [comment]
Going to head to the office to fix this one ASAP!!

**Comment by Ylianst** [comment]
Published MeshCentral v0.4.2-i with a fix for this! Thanks for the report.

**Comment by whalehub** [comment]
@Ylianst Thanks for the quick fix, I can confirm that upgrading to v0.4.2-i resolved the issue.
